src/argus_htmx/templates/tailwind/tailwind.config.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
         ...{{ themeoverride }},
     },
     safelist: [
-        "htmx-request"
+        "htmx-request",
+        { pattern: /alert-(info|success|warning|error)/ }
```

Comment:
Add in the PR description that the tailwind config needs to be rebuilt.

Reply:
yes. Ideally we'd pick up on these classes automatically but that requires specifying perhaps every `.py` file in argus_htmx as possible css class source (project dir) (and then also publishing that so that tailwind_config can automatically pick up on that, but that requires some additional discoverability